draft-irtf-cfrg-bbs-signatures.md

```diff
@@ -405,13 +412,16 @@ Procedure:
 1.  (Q_1, Q_2, H_1, ..., H_L) = create_generators(L+2)
 2.  domain = calculate_domain(PK, Q_1, Q_2, (H_1, ..., H_L), header)
 3.  if domain is INVALID, return INVALID
-4.  e_s_for_hash = serialize((SK, domain, msg_1, ..., msg_L))
-5.  if e_s_for_hash is INVALID, return INVALID
-6.  (e, s) = hash_to_scalar(e_s_for_hash, 2)
-7.  if e or s is INVALID, return INVALID
-8.  B = P1 + Q_1 * s + Q_2 * domain + H_1 * msg_1 + ... + H_L * msg_L
-9.  A = B * (1 / (SK + e))
-10. return signature_to_octets(A, e, s)
+4.  e_s_octs = serialize((SK, domain, msg_1, ..., msg_L))
+5.  if e_s_octs is INVALID, return INVALID
+6.  e_s_expand = expand_message(e_s_octs, expand_dst, expand_len * 2)
```

Comment:
`expand_message` could abort as well. Should we add a check that `e_s_expand` is not INVALID (and maybe a note that if `expand_message` aborts we assume it returns INVALID??)

Reply:
I had thought we treated it as infallible, but apparently not, so updated now.